.github/workflows/lint.yml

```diff
@@ -0,0 +1,25 @@
+name: lint
+
+on:
+  push:
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4.2.1
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4.0.4
+        with:
+          cache: npm
+          cache-dependency-path: tree-sitter-topas/package-lock.json
+          
+      - name: Install modules
+        run: npm ci
```

Comment:
Huh, I didn't realise this existed as a command, will be switching my CI over to use it too